lib/java_buildpack/framework/introscope_agent.rb

```diff
@@ -44,6 +44,8 @@ def release
           .add_system_property('com.wily.introscope.agent.agentName', agent_name(credentials))
           .add_system_property('introscope.agent.defaultProcessName', default_process_name)
 
+        java_opts.add_system_property('agentManager.credential', credential(credentials)) unless
+        credential(credentials).nil?
```

Comment:
No need for a `.nil?` check.  `nil` is boolean.

Reply:
Alright, I will remove the .nil? check.